sdk/openai/openai/tsconfig.test.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["../../../tsconfig.test.json", "./tsconfig.src.json"],
+  "compilerOptions": {
+    "lib": ["DOM"]
+  },
+  "include": ["./test", "./src"]
```

Comment:
should we have `  "include": ["./test", "./src"]` in the central config if this is the majority case?

Reply:
same reason why `"include": ["./src"]` was moving out?

Reply:
These include paths are resolved statically, so in this case, `./src` is being resolved as `${roo}/src` instead of `${root}/sdk/<service>/<lib>/src`. It doesn't cause issues because tshy override it anyway but we should still set them correctly for other tooling.

Reply:
ah right.  I think in recent version of TypeScript they added a feature for this. Let me check

Reply:
${configDir} introduced in TypeScript v5.5 https://devblogs.microsoft.com/typescript/announcing-typescript-5-5/#the-configdir-template-variable-for-configuration-files

Reply:
Awesome, thanks a lot for the pointer! I addressed it in https://github.com/Azure/azure-sdk-for-js/pull/32075/commits/f9e0856a70a4dbb171d6a65ff8bec8e523c51293